src/web/layouts/lib/stickiness.tsx

```diff
@@ -32,8 +32,6 @@ const bannerWrapper = css`
 	position: fixed !important;
 	bottom: 0;
 	${getZIndexImportant('banner')}
-	max-height: 80vh;
-	overflow: visible;
```

Comment:
This is to allow the banner to go 'fullscreen' on mobile and to get the right scrolling behaviour

Reply:
Is this comment something to worry about?

`Overflow is visible because puzzles banner needs it to render correctly`

Reply:
Very much so! I've changed that back!

Reply:
Ah nice, glad we dodged that!

I wonder if this is perhaps symptomatic of a larger issue? It'd be great to have a more centralised, controlled, solution for showing banners (or any overlay). Right now we've got multiple silos operating in the same problem space and I think we've set ourselves up for these kinds of issues. Using code comments as a way to guard against css changes affecting other teams is not a great position to be in!

Just a random thought, not a blocker for this PR

Reply:
Yeah it's a great thought. The trickiness here was because the puzzles banner can't be dismissed like the others, but instead is minimized which leaves an absolutley positioned bit of the banner in the bottom right corner (hence the need for `overflow: visible` as the banner then has height 0). In this case, we just wanted `overflow: auto` to allow the banner to be scrollable incase a user has increased their font size so much that the banner overflows.

I'm not really sure what the general solution should be. We've definitely got accessibility concerns with the banners so it'd be good to bash that out!